index.html

```diff
@@ -27,7 +27,8 @@
        <!-- Navigation Bar -->
     <div class="navbar" style="position: absolute; top: 0; right: 20px; padding: 10px; z-index: 100;; font-size: 18px;">
         <a href="index.html">Home </a>&nbsp;&nbsp;&nbsp;     
-        <a href="blog.html"> Blog</a>
+        <a href="blog.html"> Blog</a>&nbsp;&nbsp;&nbsp;
+        <a href="apizoo/">API Zoo Index</a>
```

Comment:
Since, it's more than one word "API Zoo Index" it's possible for someone who's new to read it as "Blog API Zoo Index". It'll be nice if we can find a way to show there are 3 different links. One potential is "Home | Blog | API Zoo Index"? Thoughts?

Reply:
Good catch – I added vertical bars between the titles in the navbar. Let me know what you think!